src/sentry/sentry_apps/external_requests/select_requester.py

```diff
@@ -56,11 +58,24 @@ def run(self) -> dict[str, Any]:
                     "project_slug": self.project_slug,
                     "uri": self.uri,
                     "error_message": str(e),
+                    "url": url,
                 },
             )
+            raise APIError from e
 
         if not self._validate_response(response) or not response:
-            raise APIError(
+            logger.info(
+                "select-requester.invalid-response",
+                extra={
+                    "response": response,
+                    "sentry_app_slug": self.sentry_app.slug,
+                    "install_uuid": self.install.uuid,
+                    "project_slug": self.project_slug,
+                    "uri": self.uri,
+                    "url": url,
```

Comment:
isn't uri enough for debugging here? what added value does url have?

Reply:
the final url would come in handy in checking who we are contacting, and what the final query stirng is (since for this request we tend to have a bunch of query params). 
It is somewhat redundant with the uri though, so I'd probably prefer to keep the url to debug with